ipaclient/plugins/vault.py

```diff
@@ -150,6 +152,8 @@ def decrypt(data, symmetric_key=None, private_key=None):
         except ValueError:
             raise errors.AuthenticationError(
                 message=_('Invalid credentials'))
+    else:
+        return None
```

Comment:
Same here -- probably better to return `b''`

Reply:
Same here. The change just makes ``return None`` explicit.